packages/api/src/hooks/useLoginHistory.ts

```diff
@@ -0,0 +1,35 @@
+import { useMemo } from 'react';
```

Comment:
are we using this hook anywhere?  it uses useAuthorize inside.

Reply:
this is used in line 20 of the same file, for memoizing the return value `modified_login_history`

Reply:
I mean are we using useLoginHistory.ts ? This hook have useAuthorize . Is it safe to use useAuthorize now?

Reply:
yes, it is used. i think then this too should be removed from this PR.

Reply:
@suisin-deriv can you please help to check this?